extern/launcher.c

```diff
@@ -0,0 +1,335 @@
+/*  Setuptools Script Launcher for Windows
```

Comment:
What's the license for this file? I can't include it as it is without a license header. Please add a header containing:

1. A reference to where this file is from (https://github.com/pypa/setuptools ?)

2. A copy of the License text (MIT ?)

Reply:
License/source has been added as a header. I also fixed a bug that I missed when originally making this PR. _Now_ it should be good to go :D!